### DISCUSSION TITLE
绳网现在可以发语音了

### DISCUSSION THREAD

**Discussion by IcyWtr** [share121/inter-knot#1308] [其他]
[ı||ııı||ı|ıı||ı| 9”](https://www.bilibili.com/video/BV1hq4y1s7VH/?spm_id_from=333.337.search-card.all.click&vd_source=292a736272998f2c0ba897d68fcc9f90)

**Comment by convant** [comment]
哈哈，想骗我，这玩意化成灰我都认识

**Comment by lllliufeng** [comment]
如果早知道语音也会...

**Comment by Fh52** [comment]
被骗了（

**Comment by user1-cloud** [comment]
绳网现在可以展开了 
[**展开**](https://www.bilibili.com/video/BV1s44y1k7pK/?share_source=copy_web&vd_source=722b2d6c4a0efb2b0fc1aa3c34f95a9d "展开")

**Comment by lllliufeng** [comment]
别信楼上，看我的
<details>
<summary>展开</summary>
<details>
<summary>展开</summary>
<details>
<summary>展开</summary>
<details>
<summary>展开</summary>
<details>
<summary>展开</summary>
<details>
<summary>展开</summary>
<details>
<summary>展开</summary>
<details>
<summary>展开</summary>
<details>
<summary>展开</summary>
<details>
<summary>展开</summary>

### 你被骗了
 ....................„-~~'''''''~~--„„_
..............„-~''-,::::::::::::::::::: ''-„
..........,~''::::::::',:::::::::::::::: ::::|',
.....::::::,-~'''¯¯¯''''~~--~'''¯'''-,:|
.........'|:::::|: : : : : : : : : : : ::: : |,'
........|:::::|: : :-~~---: : : -----: |
.......(¯''~-': : : :'¯°: ',: :|: :°-: :|
.....'....''~-,|: : : : : : ~---': : : :,'
...............|,: : : : : :-~~--: : ::/
......,-''\':\: :'~„„_: : : : : _,-'
__„-';;;;;\:''-,: : : :'~---~''/|
;;;;;/;;;;;;;\: :\: : :____/: :',__
;;;;;;;;;;;;;;',. .''-,:|:::::::|. . |;;;;''-„__
;;;;;;,;;;;;;;;;\. . .''|::::::::|. .,';;;;;;;;;;''-„
;;;;;;;|;;;;;;;;;;;\. . .\:::::,'. ./|;;;;;;;;;;;;|
;;;;;;;\;;;;;;;;;;;',: : :|¯¯|. . .|;;;;;;;;;,';;|
;;;;;;;;;',;;;;;;;;;;;\. . |:::|. . .'',;;;;;;;;|;;/
;;;;;;;;;;\;;;;;;;;;;;\. .|:::|. . . |;;;;;;;;|/
;;;;;;;;;;;;,;;;;;;;;;;|. .\:/. . . .|;;;;;;;;|
</details>
</details>
</details>
</details>
</details>
</details>
</details>
</details>
</details>
</details>

**Reply by Buer-Nahida** [reply]
> 别信楼上，看我的
> 展开
> 展开
> 展开
> 展开
> 展开
> 展开
> 展开
> 展开
> 展开
> 展开
> ### 你被骗了
> 
> ....................„-~'''''''~--„„_ ..............„-~''-,::::::::::::::::::: ''-„ ..........,~''::::::::',:::::::::::::::: ::::|', .....::::::,-~'''¯¯¯''''~~--~'''¯'''-,:| .........'|:::::|: : : : : : : : : : : ::: : |,' ........|:::::|: : :-~---: : : -----: | .......(¯''~-': : : :'¯°: ',: :|: :°-: :| .....'....''~-,|: : : : : : ~---': : : :,' ...............|,: : : : : :-~--: : ::/ ......,-''':: :'~„„_: : : : : _,-' __„-';;;;;:''-,: : : :'~---~''/| ;;;;;/;;;;;;;: :: : :____/: :',__ ;;;;;;;;;;;;;;',. .''-,:|:::::::|. . |;;;;''-„__ ;;;;;;,;;;;;;;;;. . .''|::::::::|. .,';;;;;;;;;;''-„ ;;;;;;;|;;;;;;;;;;;. . .:::::,'. ./|;;;;;;;;;;;;| ;;;;;;;;;;;;;;;;;;',: : :|¯¯|. . .|;;;;;;;;;,';;| ;;;;;;;;;',;;;;;;;;;;;. . |:::|. . .'',;;;;;;;;|;;/ ;;;;;;;;;;;;;;;;;;;;;. .|:::|. . . |;;;;;;;;|/ ;;;;;;;;;;;;,;;;;;;;;;;|. .:/. . . .|;;;;;;;;|

**Reply by lllliufeng** [reply]
> > 别信楼上，看我的
> > 展开
> > 展开
> > 展开
> > 展开
> > 展开
> > 展开
> > 展开
> > 展开
> > 展开
> > 展开
> > ### 你被骗了
> > ....................„-~'''''''~--„„_ ..............„-~''-,::::::::::::::::::: ''-„ ..........,~''::::::::',:::::::::::::::: ::::|', .....::::::,-~'''¯¯¯''''~~--~'''¯'''-,:| .........'|:::::|: : : : : : : : : : : ::: : |,' ........|:::::|: : :-~---: : : -----: | .......(¯''~-': : : :'¯°: ',: :|: :°-: :| .....'....''~-,|: : : : : : ~---': : : :,' ...............|,: : : : : :-~--: : ::/ ......,-''':: :'~„„_: : : : : _,-' __„-';;;;;:''-,: : : :'~---~''/| ;;;;;/;;;;;;;: :: : :____/: :',__ ;;;;;;;;;;;;;;',. .''-,:|:::::::|. . |;;;;''-„__ ;;;;;;,;;;;;;;;;. . .''|::::::::|. .,';;;;;;;;;;''-„ ;;;;;;;|;;;;;;;;;;;. . .:::::,'. ./|;;;;;;;;;;;;| ;;;;;;;;;;;;;;;;;;',: : :|¯¯|. . .|;;;;;;;;;,';;| ;;;;;;;;;',;;;;;;;;;;;. . |:::|. . .'',;;;;;;;;|;;/ ;;;;;;;;;;;;;;;;;;;;;. .|:::|. . . |;;;;;;;;|/ ;;;;;;;;;;;;,;;;;;;;;;;|. .:/. . . .|;;;;;;;;|

看[这里](https://docs.github.com/zh/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections)

**Comment by llxcx4** [comment]
很喜欢各位的一句话[doge]
BV1s44y1k7pK

**Comment by alwaylone** [comment]
hhhhhhh

**Comment by blqdswkl** [comment]
看到视频标题的一瞬间我就应激关掉标签页了

**Comment by bin-lin2** [comment]
防不胜防

**Comment by cooperatior** [comment]
泰酷辣

**Comment by shengjin114514** [comment]
一天上一当，当当不重样
